sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -104,6 +105,8 @@ func doObfuscationDetection(workDirs workDirs) (*obfuscation.AnalysisResult, err
 		return nil, fmt.Errorf("error while walking package files: %v", err)
 	}
 
+	result.PackageSignals = obfuscation.RemoveNaNs(result.PackageSignals)
```

Comment:
What is this needed for? Since PackageSignals seems to always be a NoSignals() value? Is this for future extension? If so, I wonder if this is something that can be done further down the obfuscation module so that callers don't have to be aware of needing to call this.

Reply:
Yeah the `PackageSignals` isn't currently populated with any data, but I think it's a useful thing to have in terms of the data format. The `RemoveNaNs` function is there because JSON doesn't currently support NaNs, but that's more an artifact of the serialisation method rather than anything obfuscation specific. I could potentially replace JSON serialisation with (say) Protobuf serialisation, but then this change would probably need to propagate to the resultstore code too, otherwise it will have the same problem with NaNs